### PR TITLE
Improved basic page

### PR DESCRIPTION
**Pull request by dbosen**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/thunder/thunder-distribution/blob/8.x-4.x/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.
- [ ] New features or changes are documented.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: modules/thunder_gqls/src/Plugin/GraphQL/SchemaExtension/ThunderBcLayerBasicPageBodySchemaExtension.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+namespace Drupal\thunder_gqls\Plugin\GraphQL\SchemaExtension;
+
+use Drupal\graphql\GraphQL\ResolverRegistryInterface;
+
+/**
+ * The menu schema extension.
+ *
+ * @SchemaExtension(
+ *   id = "thunder_bc_layer_basic_page_body",
+ *   name = "Back compatibility layer for the basic page body field.",
```
Comment (chrfritsch) [inline modules/thunder_gqls/src/Plugin/GraphQL/SchemaExtension/ThunderBcLayerBasicPageBodySchemaExtension.php]: I think it should be 'Backwards compatibility layer'

File: thunder.install [hunk 2]
```diff
@@ -284,3 +286,31 @@ function thunder_update_8321(): string {
...
+  ];
+
+  foreach ($newFields as $newField) {
+    $configDirectory = new FileStorage(\Drupal::service('extension.path.resolver')
+      ->getPath('profile', 'thunder') . '/config/optional');
+    $configRecord = $configDirectory->read($newField);
+    if (!FieldConfig::loadByName($configRecord['entity_type'], $configRecord['bundle'], $configRecord['field_name'])) {
+      FieldConfig::create($configRecord)->save();
+    }
+  }
```
Comment (chrfritsch) [inline thunder.install]: You could add this to your update file instead:
```
__global_actions:
  import_configs:
    - field.field.node.page.field_meta_tags
    - field.field.node.page.field_paragraphs
```

Comment (dbosen) [reply thunder.install]: Why does it not do this by itself?

Comment (dbosen) [reply thunder.install]: This is at least missing in the documentation :-)

**Review by chrfritsch — APPROVED** [review]

**Pull request merged**